WEEKLY_UPDATES.md

```diff
@@ -1,5 +1,30 @@
 # Weekly Updates
 
+## 2023-11-02
+
+Hey!
+
+This week was still mostly spent on discussing and PoC'in stable registry designs, as well as work on the S3 backend.
+
+- Current Status
+  - OpenTofu Day will be happening as a KubeCon + CloudNativeCon Europe CNCF-hosted Co-located Event, learn more [here](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/co-located-events/opentofu-day/#registration-details)!
+  - The technical steering committee is meeting today to pick a stable registry design.
```

Comment:
Add a date here so its clear to understand if you're reading this retrospectively (ie in 2 months)

Reply:
There is a date in each weekly update in the header, don't you think that's enough?